app.py

```diff
@@ -353,6 +353,35 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
             f" worth of {int(example_row['liquidable_debt_at_interval']):,} USD will be liquidated while the AMM swaps "
             f"capacity will be {int(example_row['debt_token_supply']):,} USD."
         )
+    accumulated_debt = main_chart_data[['collateral_token_price', 'liquidable_debt']].copy()
```

Comment:
I would recommend to move it out to separate function. As you can see our `main()` is so large and if we can move something out, it would be more readable